### DISCUSSION TITLE
Blocked license

### DISCUSSION THREAD

**Discussion by konrad-verelst** [aclap-dev/video-downloadhelper#1350] [Help and Q&A]
### Your question

I have a license for Firefox a few years already.
After reinstalling Windows 10, the vdhcoapp is locked.
On the downloadhelper homepage and also in this forum I read that Github provides support, but they can not help me because it is not a Github product.
What can I do?

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: de
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"locked":true,"key":"2d1c315993b84176"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
{
    "download_directory": "Q:\\! R\\! ! ! ! Neu",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "Q:\\! R\\! ! ! ! Neu"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 15,
    "successfull_dl": 42
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

When the VDH license gets locked, you're supposed to receive an E-mail containing a link you're supposed to click.  Maybe it got blocked by a spam filter.  If you didn't get that E-mail, I can't help you.  You'll have to wait for @jcfrog Jérôme (who is in France) to come in here & help you.  He does work for VDH.

**Reply by GrampaWildWilly** [reply]
I don't know what you read that led you to believe GitHub supports VDH.  That's not true & never has been.  You might have misinterpreted something that said support for VDH is provided on GitHub.  That just means that GitHub is the host for this forum.  They do not participate in the discussions here.

**Comment by jcfrog** [comment]
@konrad-verelst Hi, I guess you found our email, your license is unlocked.

**Reply by 5EJKER** [reply]
hey I have a question?

do you still have this version installed on firefox

video download helper

[9.1.0.44](http://9.1.0.44/)

because now the creator has removed the versions that worked for free and left only the old 7.0

9.1.0.44-generally there is probably no better plugin for downloading movies. it finds everything it can practically always

but recently when I transferred the profile from browser to browser

it unfortunately

something probably overwrote it

and it updated

if you have this version

could you send me the file for it. it should be called like that. "(b9db16a4-6edc-47ec-a1f4-b86292ed211d).xpi" 

and it is in the %appdata% location and there in firefox in profiles --> extension 

I also found for chrome someone saved this version so I can send it to you. but for firefox it is nowhere. 

https://www.transfernow.net/dl/Videodowloaderdlachrome
